public/components/ui_action_context_menu.tsx

```diff
@@ -38,7 +39,12 @@ export const ActionContextMenu = (props: Props) => {
   const uiActions = getUiActions();
   const actionsRef = useRef(uiActions.getTriggerActions(AI_ASSISTANT_QUERY_EDITOR_TRIGGER));
   const [open, setOpen] = useState(false);
+  const { search } = props.data;
   const [actionContext, setActionContext] = useState({
+    searchState: {
+      hasError: search.df.df$.hasError,
+      results: search.df.df$.value?.size,
```

Comment:
It's named `results`, should it return the `value`?
```suggestion
      results: search.df.df$.value,
```

Reply:
resolved, thanks!